examples/navier-stokes/densitycurrent.h

```diff
@@ -474,6 +475,7 @@ CEED_QFUNCTION(DC)(void *ctx, CeedInt Q,
 //                                       (diffussive terms will be added later)
 //
 // *******************************************************************************
+#ifdef nsplex
 CEED_QFUNCTION(IFunction_DC)(void *ctx, CeedInt Q,
                    const CeedScalar *const *in, CeedScalar *const *out) {
```

Comment:
So I can't reproduce [that warning](https://github.com/CEED/libCEED/pull/430/#issuecomment-581662639) (the `problem->apply_ifunction` no initialized) on my laptop, but I'm guessing that's due to gcc versions being different. 

My only guess as to why the warning shows is that gcc isn't reading the `#ifdef` directive correctly/well on the definition of `IFunction_DC`, which was used to get rid of the original compiler warnings (about unused functions when compiling `navierstokes.c`). Any ideas on how to resolve this?

Reply:
I can't reproduce it locally with GCC or Clang. Given that we plan on removing `navierstokes.c` and renaming `nsplex.c' as such, this issue will go away so I don't see it as a blocking issue here.